JKH/Day3/K번째수.js

```diff
@@ -0,0 +1,16 @@
+function solution(array, commands) {
+  let answer = [];
+    
+  for (let i = 0; i < commands.length; i++) {
+    let startPoint = commands[i][0] - 1;
+    let endPoint = commands[i][1];
+    let KPoint = commands[i][2] - 1;
+
+    let arr1 = array.slice(startPoint, endPoint);
+    arr1.sort((a, b) => (a - b));
+          
+    answer.push(arr1[KPoint]);
+  }
+    
+  return answer;
```

Comment:
```suggestion
  const answer = [];
    
  for (let i = 0; i < commands.length; i++) {
    const startPoint = commands[i][0] - 1;
    const endPoint = commands[i][1];
    const KPoint = commands[i][2] - 1;

    const arr1 = array.slice(startPoint, endPoint);
    arr1.sort((a, b) => (a - b));
          
    answer.push(arr1[KPoint]);
  }
    
  return answer;
```
변경되지 않는 변수는 `const` 로 선언하는게 더 좋을거 같습니다.